src/cryptography/hazmat/bindings/openssl/ec.py

```diff
@@ -188,6 +189,8 @@
 const EC_METHOD *EC_GF2m_simple_method();
 
 int EC_METHOD_get_field_type(const EC_METHOD *);
+
+const char *EC_curve_nid2nist(int nid);
```

Comment:
The names of parameters should not be included, meaning this should be written `const char *EC_curve_nid2nist(int);`

(This applies on a few other lines)